languages/javascript/stocktwits-api-client.js

```diff
@@ -0,0 +1,458 @@
+'use strict';
+
+const readline = require('readline')
+const axios = require('axios');
+
+const commandSchema = `
+Available Command Schema
+
+<action> <resource> <identifier> <value/term>
+
+get user <username or id>
+get symbol <symbol or id>
+get message <message id>
+
+list blocks <username or id>
+list mutes <username or id>
+list followers <username or id>
+list follows <username or id>
+list likes <username or id>
+list watchlist <username or id>
+
+list messages user <username or id>
+list messages symbol <symbol or id>
+list messages topic <topic or id>
+`;
+
+String.prototype.format = function() {
+
+  let currentString = this;
+
+  for (const argumentIndex in arguments) {
+    let regex = new RegExp(`\\{${argumentIndex}\\}`, 'g');
+    currentString = currentString.replace(regex, arguments[argumentIndex]);
+  }
+
+  return currentString;
+
+}
+
+function stringify(obj) {
+  return JSON.stringify(obj, null, 2);
+}
+
+const config = {
+  debug: false,
+
+  targetAPIEndpoint: 'https://api.stocktwits.com/api/2/',
+  targetAPIAuthToken: 'OAuth 3634b7a97642a6829e9db3e11199b3b149c8c818',
```

Comment:
:') It's definitely not a wise idea to add authentication tokens to source control like this, especially for a public repository. That said, I wanted to have all API calls completely functional here, and I'm not interested in setting up proper CI/CD + build configurations for this simple example code. The good news is that this account is not tied to any services that would be a critical security concern (I'm fine having the token swiped), so we're good.